### PR TITLE
Allow setting replication source via attribute

### PR DESCRIPTION
**Pull request by EMH333**

### CODE REVIEW

File: spec/replica_spec.rb [hunk 1]
```diff
@@ -50,6 +50,26 @@
...
+      context 'with source attribute' do
+        cached(:chef_run) do
+          ChefSpec::SoloRunner.new(p) do |node|
+            node.automatic['osl-mysql']['replication']['source_ip'] = '192.0.2.100'
+          end.converge(described_recipe)
+        end
+
+        before do
+          stub_search(:node, 'roles:mysql-vip').and_return([])
+        end
```
Comment (ramereth) [inline spec/replica_spec.rb:63]: In theory, this shouldn't be needed in this context. Mind trying without it?

**Review by ramereth — APPROVED** [review]
Patch bump

**Comment by EMH333** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v6.8.1.  Have a nice day!

**Pull request merged**